Dockerfile-4_0

```diff
@@ -27,12 +27,6 @@ COPY management-api-common ./management-api-common
 COPY management-api-server ./management-api-server
 RUN mvn -q -ff package -DskipTests
 
-# Download and extract Metrics Collector
-ENV MCAC_PATH /opt/metrics-collector
-RUN mkdir ${MCAC_PATH} && \
-    if test ! -e datastax-mcac-agent-${METRICS_COLLECTOR_VERSION}.tar.gz; then curl -L -O "https://github.com/datastax/metric-collector-for-apache-cassandra/releases/download/v${METRICS_COLLECTOR_VERSION}/datastax-mcac-agent-${METRICS_COLLECTOR_VERSION}.tar.gz"; fi && \
-    tar --directory ${MCAC_PATH} --strip-components 1 --gzip --extract --file datastax-mcac-agent-${METRICS_COLLECTOR_VERSION}.tar.gz
-
```

Comment:
We should not keep the MCAC enabled for too long. As long as it is running, it will do the metrics fetching even if nothing ever polls it. 

So, while we might have a working /metrics endpoint, those with large amount of tables are going to suffer as long as the MCAC is enabled, even if they never poll MCAC anymore.